build/aur/deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/env bash
+cd "$TRAVIS_BUILD_DIR/build/aur"
+
+CURRENT_VERSION=$(node -e 'process.stdout.write(require("'"${TRAVIS_BUILD_DIR}"'/package.json").version)')
+
+# Replace version in PKGBUILD with new version from package.json
+sed -i '' -e "s,pkgver=[0-9].*$,pkgver=${CURRENT_VERSION}," PKGBUILD
+
+# Obtain mksrcinfo
+wget https://www.archlinux.org/packages/community/any/pkgbuild-introspection/download/ -O pkgbuild-introspection.tar.xz
+tar -Jxf pkgbuild-introspection.tar.xz usr/bin/mksrcinfo
+PATH="$PATH:$(pwd)/usr/bin"
+
+git clone ssh://aur@aur.archlinux.org/oni.git aur
+cp PKGBUILD
+cd aur
+mksrcinfo
+
+git add PKGBUILD .SRCINFO
+git config user.email "oni" # Hmmm
```

Comment:
I guess it does make sense to use the `onivim@onivim.io` e-mail address here. Is there any other authentication we need? Maybe an ssh key? 

It looks like we can create one, and then encrypt it with Travis CI, so that it is secure but TravisCI still gets it (details here:https://oncletom.io/2016/travis-ssh-deploy/)

Reply:
@bryphe Lemme know if you want me to do anything. Don't mind owning this.

Reply:
Sure, if you'd be up for being a `co-maintainer` and setting up the encrypted private/public ssh key, that'd be a huge help!

Also, I did create an account - `onivim` with `oni@onivim.io`, but haven't set up any keys yet. @badosu - could you please add us both as co-maintainers?

Reply:
Yeah, I can do that. I'll try to set it up later tonight or tomorrow.

Reply:
> Also, I did create an account - onivim with oni@onivim.io, but haven't set up any keys yet. @badosu - could you please add us both as co-maintainers?

Done, @someguynamedmatt what's your AUR username?

Reply:
@badosu Should be the same name `someguynamedmatt`

Reply:
@someguynamedmatt Added as well.